content/en/docs/application-store/external-apps/deploy-tidb.md

```diff
@@ -0,0 +1,217 @@
+
+   ![download-yaml-file](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/download-yaml-file.PNG)
+
+4. On the **Basic Info** page, confirm the app name, app version, and deployment location. Click **Next** to continue.
+
+   ![tidb-cluster-info](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tidb-cluster-info.PNG)
+
+5. Some TiDB components require [persistent volumes](../../../cluster-administration/persistent-volume-and-storage-class/). You can run the following command to view your storage classes.
+
+   ```bash
+   kubectl get sc
+   ```
+
+   In this regard, the following storage classes will be deployed automatically by default if you install KubeSphere through QingCloud Kubernetes Engine (QKE).
+
+   ```
+   / # kubectl get sc
+   NAME                       PROVISIONER     RECLAIMPOLICY   VOLUMEBINDINGMODE   ALLOWVOLUMEEXPANSION   AGE
+   csi-high-capacity-legacy   csi-qingcloud   Delete          Immediate           true                   71m
+   csi-high-perf              csi-qingcloud   Delete          Immediate           true                   71m
+   csi-ssd-enterprise         csi-qingcloud   Delete          Immediate           true                   71m
+   csi-standard (default)     csi-qingcloud   Delete          Immediate           true                   71m
+   csi-super-high-perf        csi-qingcloud   Delete          Immediate           true                   71m
+   ```
+
+   {{< notice note >}}
+
+   The QingCloud CSI plugin implements an interface between [CSI](https://github.com/container-storage-interface/)-enabled Container Orchestrator (CO) and the storage of QingCloud. For more information about QingCloud CSI, see the [GitHub repository](https://github.com/yunify/qingcloud-csi).
+
+   {{</ notice >}}
+
+6. On the **App Config** page, change the default value of the field `storageClassName` from `local-storage` to `csi-qingcloud`. In the downloaded file, you can replace all the values directly and copy and paste the modified local file into the `values.yaml` file on the console.
+
+   ![tidb-cluster-config](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tidb-cluster-config.PNG)
+
+   {{< notice note >}}
+
+   Only the field `storageClassName` is changed to provide external persistent storage. If you want to deploy each TiDB component, such as [TiKV](https://docs.pingcap.com/tidb/dev/tidb-architecture#tikv-server) and [Placement Driver](https://docs.pingcap.com/tidb/dev/tidb-architecture#placement-driver-pd-server), to individual nodes, specify the field `nodeAffinity`.
+
+   {{</ notice >}} 
+
+7. Click **Deploy** and you can see two apps in the list as shown below:
+
+   ![tidb-cluster-app-running](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tidb-cluster-app-running.PNG)
+
+### Step 5: View TiDB cluster status
+
+1. Go to **Workloads** under **Application Workloads**, you can see all TiDB cluster Deployments are up and running.
```

Comment:
Go to **Workloads** under **Application Workloads**, you can see all TiDB cluster Deployments are up and running.
->
Go to **Workloads** under **Application Workloads**, and verify that all TiDB cluster Deployments are up and running.